src/integrationTest/resources/ccd/response/start-event.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
worth renaming this file to start-event-response but can be done later

Reply:
it's already under `response` folder. no need to be too verbose here with resources